raisinbread/instruments/eeg_acquisition_form.linst

```diff
@@ -0,0 +1,88 @@
+select{@}ecg_collected{@}ECG Collected?{@}'yes'=>'Yes'{-}'no'=>'No'
+select{@}subject_away_60cm_from_monitor{@}Subject is 60cm away from monitor?{@}'yes'=>'Yes'{-}'no'=>'No'
+textarea{@}subject_away_60cm_from_monitor_reason{@}If No, please explain the reason
+header{@}{@}<b><h3>Net information</h3></b>
+text{@}net_id{@}Net ID:
+select{@}impedance_measured_before_netting{@}Impedances measure before netting?{@}'yes'=>'Yes'{-}'no'=>'No'
+numeric{@}impedance_measure{@}Impedance measured:
+select{@}bad_channels_measured_after_netting{@}Bad channels measured after netting?{@}'yes'=>'Yes'{-}'no'=>'No'
+textarea{@}bad_channels{@}Bad channels:
+header{@}{@}<h6>About net placement:<br/> <b>Excellent</b> - meets all criteria for proper net placement.<br/> <b>Average</b> - 1-2cm shift from proper placement (1 pinky finger width distance).<br/> <b>Poor</b> - >2 cm shift from proper placement (>1 pinky finger width distance). </h6>
+select{@}net_placement{@}Net placement{@}'na'=>'n/a'{-}'excellent'=>'Excellent'{-}'average'=>'Average'{-}'poor'=>'Poor'
+textarea{@}net_placement_comment{@}Comments on net placement
+select{@}net_placement_photos_taken{@}Placement photos taken?{@}'yes'=>'Yes'{-}'no'=>'No'
+select{@}acquisition_software{@}Acquisition software used{@}'eemagine'=>'Eemagine'{-}'neuro_scan'=>'(Neuro)scan'{-}'insep'=>'InsEP'{-}'biosemi'=>'BioSemi'{-}'netstation'=>'NetStation'{-}'brain_vision_recorder'=>'Brain Vision Recorder'{-}'emotiv'=>'Emotiv'{-}'actiview605_lores'=>'ActiView605-Lores'{-}'eeprobe'=>'EEProbe'{-}'tmsi'=>'TMSi'{-}'other'=>'Other'
+text{@}acquisition_software_other{@}If other, please specify
+select{@}recording_reference{@}Recording reference used{@}'left_mastoid'=>'left mastoid'{-}'right_mastoid'=>'right mastoid'{-}'left_earlobe'=>'left earlobe'{-}'right_earlobe'=>'right earlobe'{-}'physically_linked_mastoids'=>'physically linked mastoids'{-}'physically_linked_earlobes'=>'physically linked earlobes'{-}'cz'=>'Cz'{-}'tip_nose'=>'tip of the nose'{-}'balanced_non_cephalic_reference'=>'balanced non-cephalic reference'{-}'other'=>'Other'
+text{@}recording_reference_other{@}If other, please specify
+text{@}number_active_channels{@}Number of active channels
+select{@}number_active_channels_status{@}Number of active channels{@}NULL=>''{-}'not_answered'=>'Not Answered'
+select{@}eog_channels{@}EOG channels{@}'no_eog_channels'=>'no EOG channels'{-}'two_heog_channels_only'=>'two HEOG channels only'{-}'two_veog_channels_only'=>'two VEOG channels only'{-}'two_heog_two_veog_channels'=>'two HEOG and two VEOG channels'{-}'other'=>'Other'
+text{@}eog_channels_other{@}If other, please specify
+textarea{@}artefacted_channels{@}Artefacted channels
+header{@}{@}<b><h3>EEG Acquisition</h3></b>
+select{@}bad_electrodes_present{@}Were there any bad electrodes during impedances?{@}'yes'=>'Yes'{-}'no'=>'No'
+textarea{@}bad_electrodes{@}If Yes, please register all bad electrodes
```

Comment:
@regisoc can you remind me why these questions are located down here rather than with the questions on impedance testing?

Reply:
I think it comes from the HBCD EEG acquisition form, we can put them up here to be more generic.